pkg/udf/function/uds_grpc.go

```diff
@@ -19,24 +19,24 @@ package function
 import (
 	"context"
 	"fmt"
-	clientsdk "github.com/numaproj/numaflow/pkg/sdkclient/udf/client"
 	"strconv"
 	"sync"
 	"time"
 
-	functionpb "github.com/numaproj/numaflow-go/pkg/apis/proto/function/v1"
-	functionsdk "github.com/numaproj/numaflow-go/pkg/function"
-	"github.com/numaproj/numaflow/pkg/udferr"
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/protobuf/types/known/emptypb"
 	"google.golang.org/protobuf/types/known/timestamppb"
 	"k8s.io/apimachinery/pkg/util/wait"
 
+	functionpb "github.com/numaproj/numaflow-go/pkg/apis/proto/function/v1"
```

Comment:
this doesn't look right

Reply:
means?

Reply:
`github.com/numaproj/numaflow-go` and `github.com/numaproj/numaflow` shouldn't be together since `numaflow-go` is outside of `numaflow`.

Reply:
Does your IDE has the feature to auto group? And what's the behavior? I thought packages from the same github org could stay together.